acknowledgements.rst

```diff
@@ -0,0 +1,13 @@
+.. _documentation_home:
```

Comment:
You are using the same reference in all files. This is probably copy-paste issue/typo

Reply:
That fixed it, thanks!